plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlTypeMapping.java

```diff
@@ -558,7 +643,7 @@ public void testDate()
                             DATE, "DATE '" + dateOfLocalTimeChangeBackwardAtMidnightInSomeZone.toString() + "'")
                     .execute(getQueryRunner(), session, memSqlCreateAndInsert("tpch.test_date"))
                     .execute(getQueryRunner(), session, trinoCreateAsSelect(session, "test_date"))
-                    .execute(getQueryRunner(), session, trinoCreateAsSelect(getSession(), "test_date"))
+                    .execute(getQueryRunner(), session, trinoCreateAsSelect("test_date"))
                     .execute(getQueryRunner(), session, trinoCreateAndInsert(session, "test_date"));
```

Comment:
Did you want to replace this with `trinoCreateAndInsert("test_date")` too?

Reply:
No, I just wanted to remove the unnecessary `getSession()`, but you reminded me that I should add a `trinoCreateAndInsert("test_date")` instead of replacing it, like this:

```
                    .execute(getQueryRunner(), session, memSqlCreateAndInsert("tpch.test_date"))
                    .execute(getQueryRunner(), session, trinoCreateAsSelect(session, "test_date"))
                    .execute(getQueryRunner(), session, trinoCreateAsSelect("test_date"))
                    .execute(getQueryRunner(), session, trinoCreateAndInsert(session, "test_date"))
                    .execute(getQueryRunner(), session, trinoCreateAndInsert("test_date"));
```

Reply:
While doing this change you might also want to conver fromt the loop to a DataProvider implementation. It gives better errors in case of failures. (Doesn't need to happen in this PR).

Reply:
> It gives better errors in case of failures

If I understand correctly, this is saying that we can directly see exactly which time zone test failed.

<img width="346" alt="image" src="https://user-images.githubusercontent.com/32224288/157211898-eadb42e1-e67f-49ec-8ec5-1796f60b4253.png">

> conver fromt the loop to a DataProvider implementation

And I also observed some other connector type tests with similar issues (e.g. AbstractTestOracleTypeMapping#testDate), I will file an issue to track and fix it.

Reply:
>  I should add a trinoCreateAndInsert("test_date") instead of replacing it

done

Reply:
For DataProvider let's track in separate issue. Marking this as resolved. Thanks.

Reply:
> For DataProvider let's track in separate issue.

#11373